packages/volto/news/5688.bugfix

```diff
@@ -0,0 +1 @@
+Fixed listing SSR rendering by sending subrequestId instead of id only within getAsyncData the same way it happens when calling getQueryStringResults directly. @ichim-david 
```

Comment:
```suggestion
Fixed listing SSR rendering by sending `subrequestId` instead of `id` only within `getAsyncData`, similar to calling `getQueryStringResults` directly. @ichim-david 
```